e2e/utils.ts

```diff
@@ -0,0 +1,83 @@
+import { promisify } from 'util';
+import { exec } from 'child_process';
+import { DatePeriod, Resource } from '../src/common/lib/types';
+
+const apiUrl = process.env.API_URL || 'http://localhost:8000';
```

Comment:
Could maybe default to `https://hauki-api.dev.hel.ninja/` 🤔

Reply:
Done ✅